_posts/2019-12-31-introducing-shardbox.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Introducing shardbox.org
+summary: A platform for discovering the shards ecosystem.
+author: straight-shoota
+---
+
+We're happy to announce the launch of [shardbox.org](https://shardbox.org), a database for discovering shards.
```

Comment:
I am definitely happy to have more options to discover shards and see how the ecosystem evolves. 

Since shardbox is maintained outside the crystal-lang organization I am hesitant with this opening phrase. It seems more like an effort of the crystal-lang rather than yourself for launching and maintaining it.

Reply:
Okay. Should I just change it to `I'm happy ...` to not speak as Crystal team or rephrase the entire sentence?

Reply:
As you prefer. I don't want your efforts to be shadowed by the organization. Even if all of us start to collaborate to it :-)

As a matter of consistency in the post, there are uses of the singular first person later.

Once again, I am happy for this an looking forward. So actually, feel free to add it to the community page also.

Reply:
Good idea.